cli/build.rs

```diff
@@ -481,7 +485,10 @@ fn main() {
   ts::create_compiler_snapshot(compiler_snapshot_path, &c);
 
   let cli_snapshot_path = o.join("CLI_SNAPSHOT.bin");
-  create_cli_snapshot(cli_snapshot_path);
+  let output = create_cli_snapshot(cli_snapshot_path);
+  for path in output.files_loaded_during_snapshot {
+    println!("cargo:rerun-if-changed={}", path.display())
+  }
```

Comment:
Cool, this seems cleaner, but I'm worried it might be a footgun for embedders - now it's up to embedder to print this directive. Why don't we move it to `deno_core::snapshot_util`?

Reply:
I guess the `must_use` directive is actually enough in this case